README.md

```diff
@@ -80,6 +80,11 @@ Because the browser is controlled by a script the actions might be too fast to
 watch. Use the `AGAMA_SLOWMO` variable with a delay in miliseconds between the
 actions. A reasonable value is round 50.
 
+### Perform installation
+
+To enable or disable OpenSUSE Tumbleweed installation use the `AGAMA_INSTALL` 
```

Comment:
TW is just one of the installable product, please don't mention product.

Reply:
Solved 1a30482